server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -172,24 +179,16 @@ public Repo<Manager> call(long tid, Manager manager) throws Exception {
               m = new Mutation(metadataRow);
               ServerColumnFamily.DIRECTORY_COLUMN.put(m, new Value(tabletDir));
               currentRow = metadataRow;
-              sawTabletAvailability = false;
             }
 
-            if (TabletColumnFamily.AVAILABILITY_COLUMN.hasColumns(key)) {
-              sawTabletAvailability = true;
-            }
+            // add the initial tablet availability
+            AVAILABILITY_COLUMN.put(m, TabletAvailabilityUtil.toValue(initialAvailability));
```

Comment:
This add the tablet availability for every key/value it sees in a row.  It should only add it once for the row.